src/tests/functional/plugin/conformance/subgraphs_dumper/src/utils/node.cpp

```diff
@@ -116,19 +116,17 @@ std::shared_ptr<ov::Node> clone_node(std::shared_ptr<ov::Node> node,
     for (size_t i = 0; i < node->get_input_size(); ++i) {
         std::string input_name = node_name + "_" + std::to_string(i);
         // todo: replace deprecated code && remove this w/a for constant size
-        OPENVINO_SUPPRESS_DEPRECATED_START
-        const auto constant_input = ov::get_constant_from_source(node->input(i).get_source_output());
```

Comment:
@ilya-lavrenov  @vurusovs  the API ov::get_constant_from_source() will be deprecated, ov::util::get_constant_from_source() from dev_api can not be used here, any suggestions?

Reply:
Why `ov::util::get_constant_from_source()` cannot be used?